src/Web/Telegram/API/Bot/Data.hs

```diff
@@ -105,8 +106,13 @@ instance FromJSON Contact where
 
 -- | This object represents a chat.
 data Chat = Chat
-  {
-    chat_id :: Int                -- ^ Unique identifier for this chat, not exceeding 1e13 by absolute value
+  { chat_id :: Int64
```

Comment:
First of all, thank you for fixing this! I did not pay enough attention when converting docs into haskell... :/

Recently you added `data ChatId = ChatId Integer | ChatChannel Text`. Would it be better to have it as `ChatId` or `Integer` instead of `Int64` here? 

I guess you are actually using this piece of code so I would appreciate your input.